src/js/constants/AppFormErrorMessages.js

```diff
@@ -30,32 +30,46 @@ const AppFormErrorMessages = {
   ],
   env: ["Key cannot be blank"],
   executor: ["Invalid executor format"],
-  general: [
-    "App creation unsuccessful. Check your app settings and try again.",
-    "Unknown server error, could not create or apply app.",
-    "Error:",
-    "App creation unsuccessful. Unauthorized access."
-  ],
   instances: ["Instances must be a non-negative Number"],
   labels: ["Key cannot be blank"],
   mem: ["Memory must be a non-negative Number"],
-  ports: ["Ports must be a comma-separated list of numbers"],
-  appLocked: ["Error: App is currently locked by one or more deployments. " +
+  ports: ["Ports must be a comma-separated list of numbers"]
+});
+
+const generalErrors = Object.freeze({
+  appCreation:
+    "App creation unsuccessful. Check your app settings and try again.",
+  appLocked: "App is currently locked by one or more deployments. " +
     "Pressing the button again will forcefully change and deploy " +
-    "the new configuration."],
+    "the new configuration.",
+  unknownServerError: "Unknown server error, could not create or apply app.",
+  unauthorizedAccess: "App creation unsuccessful. Unauthorized access.",
+  errorSuffix: "Error:"
+});
 
-  // Those are mappings to server side error messages
-  "error.path.missing": ["Please provide a path"],
+const serverResponseMappings = Object.freeze({
+  "error.path.missing": "Please provide a path"
+});
 
-  getMessage: function (fieldId, index = 0) {
-    if (this[fieldId] != null && this[fieldId][index] != null) {
-      return this[fieldId][index];
+const AppFormErrorMessages = {
+  getFieldMessage: function (fieldId, index = 0) {
+    if (applicationFieldValidationErrors[fieldId] != null &&
+        applicationFieldValidationErrors[fieldId][index] != null) {
+      return applicationFieldValidationErrors[fieldId][index];
+    }
+    return "Undefined error";
+  },
+
+  getGeneralError: function (key) {
```

Comment:
Let's rewrite these for consistency. My proposal:

```
AppFormErrorMessages
  .getFieldValidationError()
  .getGenericError()
  .getServerResponseError()
```
We have too much confusion otherwise, with `error`, `message`, `lookup`, `get` and so on...